### PR TITLE
Release v4.4.19

### PR DESCRIPTION
**Pull request by dvaldivia**
Signed-off-by: Daniel Valdivia <18384552+dvaldivia@users.noreply.github.com>

### CODE REVIEW

**Review by cniackz — APPROVED** [review]
LGTM as long as all tests are passing here.

File: bundles/certified-operators/4.4.19/manifests/minio-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -489,7 +489,7 @@ spec:
                                 - minio-operator
                         topologyKey: kubernetes.io/hostname
                 containers:
-                  - image: quay.io/minio/operator:v4.4.18
+                  - image: quay.io/minio/operator:4.4.19
```
Comment (harshavardhana) [inline bundles/certified-operators/4.4.19/manifests/minio-operator.clusterserviceversion.yaml:492]: There is no such thing as 

```
docker pull quay.io/minio/operator:4.4.19
Error response from daemon: manifest for quay.io/minio/operator:4.4.19 not found: manifest unknown: manifest unknown
```

**Review by harshavardhana — CHANGES_REQUESTED** [review]

**Review by Alevsk — CHANGES_REQUESTED** [review]
Can we include this PR on this release https://github.com/minio/operator/pull/1126 ?

File: bundles/certified-operators/4.4.19/manifests/minio-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (dvaldivia) [reply bundles/certified-operators/4.4.19/manifests/minio-operator.clusterserviceversion.yaml:492]: But there will be after you push it :P @harshavardhana

Comment (harshavardhana) [reply bundles/certified-operators/4.4.19/manifests/minio-operator.clusterserviceversion.yaml:492]: no 

```
docker pull quay.io/minio/operator:4.4.18
Error response from daemon: manifest for quay.io/minio/operator:4.4.18 not found: manifest unknown: manifest unknown
```

Comment (cniackz) [reply bundles/certified-operators/4.4.19/manifests/minio-operator.clusterserviceversion.yaml]:
```sh
$ docker pull quay.io/minio/operator:v4.4.18
v4.4.18: Pulling from minio/operator
Digest: sha256:9d9e05924e7d9faf936f71f325559066ef146f6b3360e43b608f57f8d5611a3c
Status: Image is up to date for quay.io/minio/operator:v4.4.18
quay.io/minio/operator:v4.4.18
```

**Pull request merged**